### ISSUE TITLE
Memory issue

### ISSUE THREAD

**Issue by GRIGORR** [google-research/bert#375]
I have relatively small descriptions (up to 40 tokens) and I want to change some parameters from config.json (hidden size, max_position_emb, intermediate size, etc.) to reduce memory consumption, but it results in shape mismatch error. Can it be fixed?

**Comment by xwzhong** [comment]
yes, if you want to change the model structure parament(hidden size, max_position_emb, intermediate size) you need rerun run_pretaining.py with your paraments, but, you can change the max_seq_length to decrease memory usage.
